Ride List Info Type Additional Options

thyeggman opened the pull request:
String 5715 is for a different change... I hope it won't screw things up by adding it here.

pizza2004 commented:
You appear to be somewhat out of date.  It might be good to rebase and redo the strings, since many have been added that you're trying to use.

duncanspumpkin commented:
@thyeggman you've done a merge. We prefer a rebase it makes the history nice and clean.

thyeggman commented:
Sorry... I don't know what I did wrong.  I did 'git rebase develop [mybranch]' and then resolved the conflicts.... Is that not right?  I'll try to do it more cleanly in the future.

janisozaur commented on data/language/english_uk.txt:
```diff
@@ -2364,7 +2364,7 @@ STR_2356    :{WINDOW_COLOUR_2}Rides inspected: {BLACK}{COMMA16}
 STR_2358    :Units
 STR_2359    :Real Values
-STR_2360    :Display Resolution:
+STR_2360    :{WINDOW_COLOUR_2}Display Resolution:
```
you reverted a change here

duncanspumpkin commented:
its the other way round. First reset your develop to origin. `git reset --hard origin/develop` then checkout your branch `git checkout mybranch` then rebase develop `git rebase develop`

thyeggman commented:
Let me know if I need to do anything else or if this just needs to pass the checks and be reviewed before it can be merged.

duncanspumpkin commented on data/language/english_uk.txt:
```diff
@@ -2763,8 +2763,8 @@ STR_2753    :Mowed grass
 STR_2756    :Remove litter
-STR_2757    :<not used anymore>
-STR_2758    :<not used anymore>
+STR_2757    :Force Sun
```
Not sure what's happened here this shouldn't be reverted

thyeggman commented:
I can't figure out what the hell I am doing wrong.  There are way too many files that changed.  I rebased off of dev and fixed the literal 2 lines in string_ids.txt and now everything is different....

danidoedel commented on data/language/english_uk.txt:
```diff
@@ -4002,7 +4002,6 @@ STR_5660    :Modify Groups
 STR_5661    :Set Player Group
 STR_5662    :N/A
 STR_5663    :Clear Landscape
-STR_5664    :Cheat
```
You reverted a change here.

danidoedel commented on data/language/english_uk.txt:
```diff
@@ -4039,8 +4038,20 @@ STR_5732    :Anisotropic
 # tooltip for tab in options window
 STR_5734    :{SMALLFONT}{BLACK}Rendering
-STR_5735    :Network Status
-STR_5736    :Player
```
And you reverted two changes here.

thyeggman replied:
There has got to be a better way to implement these strings.  This file has been the single largest headache that I've had trying to get this merged. Now I have to go redo TWO files AGAIN because a change happened after the first rebase I did.  I have to go and move all my string numbers down because a different change was merged before mine.  Does this happen for almost literally every merged PR that touches this file?  

I mean, I'll fix it, but it seems like this is way more of a headache than it needs to be.

pizza2004 replied:
I don't think there's usually this many additions all at once, but I think that the language files in general will probably end up needing to be refactored once the entire game is disassembled.  It's possible that's already the plan, I don't remember for sure.

duncanspumpkin replied:
@thyeggman yeah we know it is annoying but it doesn't happen that often and it doesn't exactly take too long to change. There was talk a little while ago about reserving numbers which you can do by just doing a quick PR with which ones you want reserved but I haven't seen anyone do that for quite a while.

thyeggman replied:
I'll keep that in mind, it's a good idea once I know what strings I need.  I somehow broke my dev installation of ORCT so that's why this is taking a while.

IntelOrca commented:
Looks fine, only think I would consider changing is adding another string so you have age in singular and plural form. This is done with everything else in the game, so it would be inconsistent to not do it for this.

IntelOrca commented:
Merged: 95d4bd6e603b73e65eecd49513d3f35819ad21ed

thyeggman commented:
Thanks for merging this, my apologies for not staying on top of it.
